src/scales/scale.time.js

```diff
@@ -652,7 +652,7 @@ class TimeScale extends Scale {
 			tickOpts.callback
 		]);
 
-		return formatter ? formatter(label, index, ticks) : label;
+		return formatter ? formatter(label, index, ticks, me) : label;
```

Comment:
Oh, looks like we are calling the callback inconsistently. Should probably make it consistent.

Reply:
Yeah, there's multiple things I don't like about this. One of the biggest is that the first parameter is different here. I'd noticed while working on this. It's not that easy to change though and a bit outside the scope of this PR, so probably better handled separately